build.gradle

```diff
@@ -41,10 +41,11 @@ allprojects {
     compile 'net.savantly:graphite-client:1.1.0-RELEASE'
     compile 'com.timgroup:java-statsd-client:3.0.1'
     compile 'com.signalfx.public:signalfx-codahale:0.0.47'
-    compile group: 'org.apache.kafka', name: 'kafka_2.12', version: '2.4.0'
-    compile group: 'org.apache.kafka', name: 'kafka-clients', version: '2.3.1'
+    compile group: 'org.apache.kafka', name: 'kafka_2.13', version: '2.8.2'
```

Comment:
Scala 2.13 is not supported in LI internal environment.  We should use the 2.12 variany